awx_collection/tests/integration/targets/tower_organization/tasks/main.yml

```diff
@@ -12,6 +12,8 @@
 - name: "Create a new organization"
   tower_organization:
     name: "{{ org_name }}"
+    galaxy_credentials:
+      - Ansible Galaxy
   register: result
```

Comment:
I'd also like to put this in `EXAMPLES` in `awx_collection/plugins/modules/tower_organization.py`

because that is what will go live at https://docs.ansible.com/ansible/2.10/collections/awx/awx/tower_organization_module.html

Reply:
Agreed.